docs/src/main/asciidoc/README.adoc

```diff
@@ -1,15 +1,21 @@
-# OkHttpClient integration with Spring Cloud
+= Spring Cloud Square
+
+== Quick Start
+
+include::quickstart.adoc[leveloffset=+1]
+
+== OkHttpClient integration with Spring Cloud
 
 This module supplies integration with Square's [`OkHttpClient`](https://square.github.io/okhttp/) and  [Spring Cloud LoadBalancer](https://github.com/spring-cloud/spring-cloud-commons/spring-cloud-loadbalancer).
 
 An application interceptor is added to the `OkHttpClient` created via autoconfiguration which resolves the scheme, host and port from Spring Cloud LoadBalancer and rewrites the url.
 
-By supporting `OkHttpClient`, it enables Square's [Retrofit](https://square.github.io/retrofit/) to use  Spring Cloud LoadBalancer as well.
+By supporting `OkHttpClient`, it enables Square's [Retrofit](https://square.github.io/retrofit/) to use Spring Cloud LoadBalancer as well.
 
 See [`OkHttpLoadBalancerInterceptorTests`](https://github.com/spring-cloud-incubator/spring-cloud-square/blob/master/spring-cloud-square-okhttp/src/test/java/org/springframework/cloud/square/okhttp/loadbalancer/OkHttpLoadBalancerInterceptorTests.java) for Spring Cloud LoadBalancer samples.
 
-# Spring WebClient
+== Spring WebClient
 
-Support was also added for Spring WebClient. This implements an `okhttp3.Call.Factory` that uses `WebClient` under the covers. This provides a fully non-blocking shim instead of using okhttp3.
+Support was also added for Spring WebClient.This implements an `okhttp3.Call.Factory` that uses `WebClient` under the covers.This provides a fully non-blocking shim instead of using okhttp3.
```

Comment:
Add a space after "WebClient" and after "covers.", reversing the previous change.

Add backticks around `okhttp3`, to render it as code.